### PR TITLE
KickAssAnime | Modified Service Name & Other Changes, Added Backup for Movie Detection

### PR DESCRIPTION
**Pull request by StrikerFRFX**
Title lol

Changed the Logo, Changed Service Name, Description, Added Backup (Mirror) URL

Also fixed some dumb code

Version bumped ofc.

### CODE REVIEW

File: websites/K/Kickassanime/presence.ts [hunk 1]
```diff
@@ -56,6 +56,16 @@ function checkIfMovie() {
     : nextEpisodeElement !== null && previousEpisodeElement !== null
     ? (isMovie = false)
     : (isMovie = true);
+
+  !isMovie
+    ? presence
+        .getPageletiable("appData")
+        .catch((appData) =>
```
Comment (ririxidev) [inline websites/K/Kickassanime/presence.ts:63]: eee catch? so you catch error then check anime name? I really don't understand it, could you explain it? Thanks.

File: websites/K/Kickassanime/dist/metadata.json [hunk 2]
```diff
@@ -3,13 +3,16 @@
     "name": "Striker",
     "id": "215509157837537280"
   },
-  "service": "Kickassanime",
+  "service": "KickAssAnime",
```
Comment (ririxidev) [inline websites/K/Kickassanime/dist/metadata.json:6]: If you capitalize name etc you should do that  in folder name too 😄

**Review by ririxidev — CHANGES_REQUESTED** [review]
You should check out thing that confuses me and metadata thing :)

Thanks,
ririxi (Contact: mail@ririxi.dev, ririxi#2721)

File: websites/K/Kickassanime/presence.ts (hunk 1 above)
Comment (StrikerFRFX) [reply websites/K/Kickassanime/presence.ts:63]: Wait. I think I might have fucked this XD

File: websites/K/Kickassanime/dist/metadata.json (hunk 2 above)
Comment (StrikerFRFX) [reply websites/K/Kickassanime/dist/metadata.json:6]: I changed it :P didn't have time to commit and check out tho

File: websites/K/Kickassanime/presence.ts (hunk 1 above)
Comment (ririxidev) [reply websites/K/Kickassanime/presence.ts:63]: P.S. ``.then`` instead of ``.catch``

File: websites/K/Kickassanime/presence.ts [hunk 3]
```diff
@@ -56,6 +56,16 @@ function checkIfMovie() {
...
+          appData.anime.types[0].name == "Movie" ? (isMovie = true) : false
```
Comment (Bas950) [inline websites/K/Kickassanime/presence.ts:64]:
```suggestion
          isMovie = appData.anime.types[0].name == "Movie" ? true : false
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by StrikerFRFX** [comment]
Not sure how to change the folder name since after changing locally nothin happens 
image

**Comment by SlowLife1661** [comment]
> Not sure how to change the folder name since after changing locally nothin happens
> image

Either change it to something diff and then change it back again or use https://git-scm.com/docs/git-mv

File: websites/K/KickAssAnime/presence.ts [hunk 4]
```diff
@@ -56,6 +56,21 @@ function checkIfMovie() {
...
+    ? presence
+        .getPageletiable("appData")
+<<<<<<< HEAD
+        .then((appData) =>
+          appData.anime.types[0].name == "Movie" ? (isMovie = true) : false
+=======
+        .catch((appData) =>
+          isMovie = appData.anime.types[0].name == "Movie" ? true : false
+>>>>>>> kickassanime
+        )
```
Comment (ririxidev) [inline websites/K/KickAssAnime/presence.ts:70]: you broke it...

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @StrikerFRFX!
You should check out thing(s) that I requested to change in my review. Please look at them!

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

File: websites/K/KickAssAnime/presence.ts (hunk 4 above)
Comment (StrikerFRFX) [reply websites/K/KickAssAnime/presence.ts:70]: h o w

Comment (StrikerFRFX) [reply websites/K/KickAssAnime/presence.ts:70]: I hate this

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy errors, and upload new proof of it working afterwords.

**Comment by StrikerFRFX** [comment]
a7YPYDcmuq
vhpLV1hjcL

these are the screenshots you are looking for

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @StrikerFRFX!
You should check out thing(s) that I requested to change in my review. Please look at them!
 - Update name on Discord Developers Portal. 

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

**Comment by StrikerFRFX** [comment]
> Heya @StrikerFRFX!
> You should check out thing(s) that I requested to change in my review. Please look at them!
> 
> * Update name on Discord Developers Portal.
> 
> Thanks,
> **ririxi**. PreMiD Reviewer Team.
> Contact: [mail@ririxi.dev](mailto:mail@ririxi.dev), ririxi#2721

Already done

**Review by ririxidev — APPROVED** [review]
LGTM!

File: websites/K/KickAssAnime/presence.ts [hunk 5]
```diff
@@ -56,6 +56,16 @@ function checkIfMovie() {
     : nextEpisodeElement !== null && previousEpisodeElement !== null
     ? (isMovie = false)
     : (isMovie = true);
+
+  !isMovie
+    ? presence
+        .getPageletiable("appData")
+        .then((appData) => {
+          appData.anime.types[0].name == "Movie" ? (isMovie = true) : false;
```
[the remark this answers is not in the document]
Comment (StrikerFRFX) [reply websites/K/KickAssAnime/presence.ts:64]: kewl, tnx

[the remark this answers is not in the document]
Comment (Bas950) [reply websites/K/KickAssAnime/presence.ts:64]:
```suggestion
          isMovie = appData.anime.types[0].name == "Movie";
```
Shouldnt it be this? or...?

**Review by ririxidev — APPROVED** [review]
LGTM

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**